app/views/spaces/edit.html.erb

```diff
@@ -10,7 +10,7 @@
 
 <fieldset>
   <header>
-    <h3>Rooms</h3>
+    <h3>Sections</h3>
```

Comment:
I don't know if we have a pattern yet for how we localize headers of pages but I wish we did.